.github/ISSUE_TEMPLATE/chore_task.md

```diff
@@ -0,0 +1,26 @@
+---
+name: 🧹Chore or task
+about: Identify a necessary task to be addressed.
+title: ''
+labels: 'kind/chore'
+assignees: ''
+---
+**Current Behavior**
+<!-- A brief description of what the current circumstance is. -->
+
+**Desired Situation**
+<!-- A brief description of the necessary action to take. -->
+
+---
+[Optional] **Alternatives**
+<!-- A brief description of any alternative solutions or features you've considered. -->
+
+[Optional] **Additional context**
+<!-- Add any other context or screenshots about the chore or task here. -->
+
+---
+**Contributor Resources**
+
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
```

Comment:
@amino19 Refer this, https://github.com/layer5io/gowrk2/pull/15/files#r663357864